src/cockpit/389-console/src/monitor.jsx

```diff
@@ -643,10 +644,7 @@ export class Monitor extends React.Component {
                 })
                 .fail(err => {
                     const errMsg = JSON.parse(err);
-                    this.props.addNotification(
-                        "error",
-                        cockpit.format("Error detecting DB implementation type - $0", errMsg.desc)
-                    );
+                    console.log("getDBEngine - Error detecting DB implementation type - $0", errMsg.desc);
```

Comment:
`$0` won't work with `console.log`, so it should be this, probably:

```suggestion
                    console.log("getDBEngine - Error detecting DB implementation type -", errMsg.desc);
```